playbooks/ha_log_cfg_sync.yaml

```diff
@@ -65,6 +65,13 @@
         content: "{{ rsync_password }}"
         mode: 0600
 
+    - name: Teardown original slave rsyncd(when slave master switched)
+      shell: |
+        set -xe
+        systemctl disable rsyncd.service || true
+      args:
+      executable: /bin/bash
```

Comment:
need a indent at the head of L73(event this is ok, not a good habit :) )

Reply:
oh, yes...